reV/supply_curve/supply_curve.py

```diff
@@ -256,30 +269,51 @@ def _merge_sc_trans_tables(cls, sc_points, trans_table,
             This is performed by merging left with trans_table, so there may be
             rows with nan sc_gid.
         """
-        trans_table = cls._parse_trans_table(trans_table)
+        if isinstance(trans_table, (list, tuple)):
+            trans_sc_table = []
+            for table in trans_table:
+                trans_sc_table.append(cls._merge_sc_trans_tables(
+                    sc_points, table, sc_cols=sc_cols))
 
-        if isinstance(sc_cols, tuple):
-            sc_cols = list(sc_cols)
+            trans_sc_table = pd.concat(trans_sc_table)
+        else:
+            trans_table = cls._parse_trans_table(trans_table)
+
+            if compare_cap and 'max_cap' in trans_table:
+                trans_sc_table = []
+                for cap, table in trans_table.groupby('max_cap'):
+                    mask = sc_points['capacity'] <= cap
+                    trans_sc_table.append(cls._merge_sc_trans_tables(
+                        sc_points.loc[mask].copy(),
+                        table,
+                        compare_cap=False,
+                        sc_cols=sc_cols))
```

Comment:
I think i understand what this is doing. Is it okay if there are duplicates afterward? Is there any check for duplicates if not?

Reply:
Good catch, this is what I realized when we were chatting earlier. I've had Bannister add in a min_cap column and updated the code to make the bins non-overlapping